StreamChatClient_v3/ChatClient.swift

```diff
@@ -151,80 +142,264 @@ public class Client<ExtraData: ExtraDataTypes> {
     private var urlSessionConfiguration: URLSessionConfiguration {
         let config = URLSessionConfiguration.default
         config.waitsForConnectivity = true
-        config.httpAdditionalHeaders = authSessionHeaders
+        config.httpAdditionalHeaders = sessionHeaders
         return config
     }
     
-    /// Stream-specific request auth headers.
-    private var authSessionHeaders: [String: String] {
+    /// Stream-specific request headers.
+    private var sessionHeaders: [String: String] {
         var headers = [
             "X-Stream-Client": "stream-chat-swift-client-\(SystemEnvironment.version)",
             "X-Stream-Device": SystemEnvironment.deviceModelName,
             "X-Stream-OS": SystemEnvironment.systemName,
             "X-Stream-App-Environment": SystemEnvironment.name
```

Comment:
This is not really relevant anymore. Backend don't use additional headers. We should change it in this way:
```
"X-Stream-Client": "stream-chat-swift-client-\(SystemEnvironment.version)|device=\(SystemEnvironment.deviceModelName)|os=\(SystemEnvironment.systemName)..."
```

Reply:
So why do we still use it in v2? 🙄  @buh, where can I find more info about this?

Reply:
Unfortunately you have to ask Marcelo directly about additional headers.
I planned to change that in v2, because now it's useless.

Reply:
I got confirmation from Marcel to do in this way:
`"X-Stream-Client": "stream-chat-swift-client-1.0.0|iPhone10|iOS13.6|production"`

Reply:
please check 1236cc92a9f211ca970a451c985790c0e52595e9

Reply:
1. Actually as I understand no need labels. You can pass just values splitter by `|`
2. I think it's good to add `SystemEnvironment.name` too

Reply:
I removed the labels and added `name`